.travis.yml

```diff
@@ -80,8 +80,7 @@ matrix:
     - ARROW_TRAVIS_JAVA_BUILD_ONLY=1
     # ARROW-2999 Benchmarks are disabled in Travis CI for the time being
     # - ARROW_TRAVIS_PYTHON_BENCHMARKS=1
-    - CC="clang-6.0"
-    - CXX="clang++-6.0"
+    - MATRIX_EVAL="CC=gcc-4.9 && CXX=g++-4.9"
```

Comment:
Why MATRIX_EVAL?

Reply:
because CC and CXX get exported before Travis re-exports them according to the build matrix (see the JIRA issue printout).

Reply:
(the other build matrix entries do it the same way)

Reply:
Uh, I see, thanks.

Reply:
FYI: We can find the `MATRIX_EVAL` technique in Travis CI document: https://docs.travis-ci.com/user/languages/c/#gcc-on-linux